src/main/java/org/mule/tools/apikit/model/Properties.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.apikit.model;
+
+import java.util.Map;
+
+public class Properties {
+
+  private final String format;
```

Comment:
I think format should have a default value. Please, add a test where it is not provided.